hack/build-single-manifests.sh

```diff
@@ -21,3 +21,12 @@ kustomize build ./deploy/manifests/enterprise-k8s \
 kustomize build ./deploy/manifests/enterprise \
   > deploy/single/all-in-one-postgres-enterprise.yaml
 
+# Kong Dev Config
+cat  ./deploy/manifests/base/namespace.yaml \
+     ./deploy/manifests/base/custom-types.yaml \
+     ./deploy/manifests/base/rbac.yaml \
+     ./deploy/manifests/base/service.yaml \
+     ./deploy/manifests/base/custom-server-block.yaml \
+     ./deploy/manifests/base/validation-service.yaml \
+     ./deploy/manifests/dev-env/kong-ingress-dbless.yaml \
```

Comment:
One last change, let's keep dev-env ` YAMLs inside `hack` directory only.
Users frequently navigate around in the `deploy` directory and we don't want them to confuse them (more).

There will be more dev-env files in future for DB-mode as well.

Reply:
Cool. I re-shuffled the directories a bit. I think it's a bit cleaner this way.